ivy_tests/test_ivy/test_frontends/test_torch/test_spectral_ops.py

```diff
@@ -0,0 +1,33 @@
+import ivy_tests.test_ivy.helpers as helpers
+from hypothesis import strategies as st
+from ivy_tests.test_ivy.helpers.testing_helpers import handle_frontend_test
+from ivy_tests.test_ivy.test_frontends.test_torch.test_tensor import _requires_grad
+
+
+@handle_frontend_test(
+    fn_tree="torch.hann_window",
+    dtype_and_x=helpers.dtype_and_values(available_dtypes=helpers.get_dtypes("int")),
```

Comment:
the `kind` arg for `helpers.get_dtypes` does not take `int`. It should be `integer`. (You can check the docstring to see all the values it takes)

Reply:
got it.. thanks